assets/javascripts/discourse/components/tc-message.js

```diff
@@ -32,6 +32,23 @@ export default Component.extend({
     );
   },
 
+  didInsertElement() {
+    this._super(...arguments);
+    if (!this.currentUser) {
+      return;
+    }
+    this.element
+      .querySelector(".tc-text")
+      .querySelectorAll(".mention")
+      .forEach((node) => {
+        const mention = node.textContent.trim().substr(1);
+        const highlightable = [this.currentUser.username, "here", "all"];
```

Comment:
maybe have here and all in some constant?

Reply:
sure, done!